en/config/autotune.md

```diff
@@ -0,0 +1,223 @@
+
+:::warning
+Auto-tuning is performed while flying.
+The airframe must fly well enough handle moderate disturbances, and should be closely attended:
+- Test that your vehicle is [stable enough for autotuning](#pre-tuning-test).
+- Be ready to abort the autotuning process by moving remote control sticks.
+- Verify that the vehicle flies well after tuning.
+:::
+
+
+## Overview
+
+PX4 uses [PID controllers](../flight_stack/controller_diagrams.md) (rate, attitude, velocity, and position) to calculate the outputs required to move a vehicle from its current estimated state to match a desired setpoint.
+The controllers must be well tuned in order to get the best performance out of a vehicle.
+In particular, a poorly tuned rate controller results in less stable flight in all modes, and takes longer to recover from disturbances.
+
+Generally if you use an [airframe configuration](../config/airframe.md) that is similar to your vehicle then the vehicle will be able to fly.
+However unless the configuration precisely matches your hardware you should tune the rate and attitude controllers.
+Tuning the velocity and position controllers is less important because they are less affected by vehicle dynamics, and the default tuning configuration for a similar airframe is often sufficient.
+
+Autotuning provides an automatic mechanism to tune the rate and attitude controllers.
+It can be used to tune fixed wing and multicopter vehicles, and VTOL vehicles when flying as a multicopter or as a fixed wing (transition between modes must be manually tuned).
+In theory it should work for other vehicle types that have a rate controller, but currently only the above types are supported.
+
+Automatic tuning works well for the multicopter and fixed wing vehicle configurations supported by PX4, provided the frame is not too flexible
+(see [below for more information](#does-autotuning-work-for-all-supported-airframes)).
+
+The vehicle must be flying in [Hold mode](../flight_modes/hold.md), [Position mode](../flight_modes/position_mc.md), or [altitude mode](../flight_modes/altitude_mc.md).
+The flight stack will apply a small disturbance to the vehicle in each axis and then attempt to calculate the new tuning parameters.
+For fixed wing vehicles the new tuning is applied in-air by default, after which the vehicle tests the new settings and reverts the tuning if the controllers are not stable.
+For multicopter, the vehicle lands and applies the new tuning parameters after disarming; the pilot is expected to then take off carefully and test the tuning.
+
+The tuning process takes about 40 seconds ([between 19 and 68 seconds](#how-long-does-autotuning-take)).
+The default behaviour can be configured using [parameters](#parameters).
+
+:::note
+For instructions on using the manual PID tuning see:
+- [Multicopter PID Tuning Guide](../config_mc/pid_tuning_guide_multicopter_basic.md) (Manual/Simple)
```

Comment:
I am assuming these are still useful. BUT we do need someone else - @bresch to go through and update. Unless this is just a matter of adding a heading "you can probably do this with autotune" at the top (can I do this)?

For advanced tuning, what is still useful? Just the stuff around airmode?

The point is, we want to point to what you do next, and tell people what they can now ignore.